splice_machine/src/test/java/com/splicemachine/derby/impl/sql/execute/operations/TernaryOperationIT.java

```diff
@@ -213,9 +213,51 @@ public void testTRIM() throws Exception {
             iCell = rs.getInt(1);
         }
         Assert.assertEquals("Wrong trim result", 4, iCell);
+    }
+
+    @Test
+    public void testTRIMWithAlias() throws Exception {
+        ResultSet rs;
+        int iCell = 0;
+        String sql = "select count(*) from ( select trim(b) from b ) tab";
+        rs = methodWatcher.executeQuery(sql);
+        rs.next();
+        iCell = rs.getInt(1);
+        Assert.assertEquals("Wrong trim result", 5, iCell);
+
+        sql = "select count(*) from ( select trim (b.b) from b ) tab";
+        rs = methodWatcher.executeQuery(sql);
+        rs.next();
+        iCell = rs.getInt(1);
+        Assert.assertEquals("Wrong trim result", 5, iCell);
+
+        sql = "select count(*) from ( select trim (b 'a' from c) from a b ) tab";
+        rs = methodWatcher.executeQuery(sql);
+        rs.next();
+        iCell = rs.getInt(1);
+        Assert.assertEquals("Wrong trim result", 4, iCell);
+
+        sql = "select count(*) from ( select trim (b.c) from a b ) tab";
+        rs = methodWatcher.executeQuery(sql);
+        rs.next();
+        iCell = rs.getInt(1);
+        Assert.assertEquals("Wrong trim result", 4, iCell);
+
+        sql = "select count(*) from ( select trim (l.c) from a l ) tab";
+        rs = methodWatcher.executeQuery(sql);
+        rs.next();
+        iCell = rs.getInt(1);
+        Assert.assertEquals("Wrong trim result", 4, iCell);
+
+        sql = "select count(*) from ( select trim (t.c) from a t ) tab";
+        rs = methodWatcher.executeQuery(sql);
+        rs.next();
+        iCell = rs.getInt(1);
+        Assert.assertEquals("Wrong trim result", 4, iCell);
 
```

Comment:
Can you add the following two test cases too:
1) negative test case:
select trim (leading.c) from a as leading;
2) positive test case:
select trim ("leading".c) from a as "leading";

Reply:
sure, added these tests.

Reply:
Thanks @changli6